components/financial_assistance/db/seedfiles/translations/en/dc/financial_assistance.rb

```diff
@@ -105,8 +105,8 @@
   "en.faa.other_incomes.cancel" => "Cancel",
   "en.faa.other_incomes.save" => "Save",
   "en.faa.other_incomes.remove" => "Remove",
-  "en.faa.other_incomes.start.date" => "Start Date",
-  "en.faa.other_incomes.end.date" => "End Date",
+  "en.Faa.other_Incomes.start_date" => "Start Date",
```

Comment:
I think the problem will be fixed just by changing ```start.date``` to ```start_date```. We might want to keep everything lowercase to match the rest of the translation files.

Reply:
absolutely.